tests/test_gossip.py

```diff
@@ -876,13 +882,34 @@ def test_gossip_notices_close(node_factory, bitcoind):
     wait_for(lambda: len(l1.rpc.listnodes()['nodes']) == 2)
     l1.rpc.disconnect(l2.info['id'])
 
+    # Grab channel_announcement from io logs (ends in ')
+    channel_announcement = l1.daemon.is_in_log('\[IN\] 0100').split(' ')[-1][:-1]
+    channel_update = l1.daemon.is_in_log('\[IN\] 0102').split(' ')[-1][:-1]
+    node_announcement = l1.daemon.is_in_log('\[IN\] 0101').split(' ')[-1][:-1]
```

Comment:
Are those `[:-1]` to strip a newline? If yes, please use `.strip()` instead, that's more explicit.